com.github.jkotra.unlockr.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.jkotra.unlockr
+runtime : org.gnome.Platform
+runtime-version : '42'
+sdk : org.gnome.Sdk
+command: unlockr
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this?

Reply:
according to [docs](https://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions):
`--socket=wayland` for wayland 
`--socket=fallback-x11` if wayland is not available.
`--share=ipc` required by x11.

please suggest if I'm wrong.

Reply:
`- --talk-name=org.freedesktop.Flatpak` is the only one I was asking about here.

Reply:
removed now.